src/core/components/ErrorPage/NotFound/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import { oneLine } from 'common-tags';
+import { compose } from 'redux';
+import NestedStatus from 'react-nested-status';
+
+import translate from 'core/i18n/translate';
+
+
+export class NotFoundBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+    status: PropTypes.number.isRequired,
+  }
+
+  static defaultProps = {
+    status: 404,
+  }
+
+  render() {
+    const { i18n, status } = this.props;
+
+    return (
+      <NestedStatus code={status}>
+        <div className="ErrorPage NotFound">
+          <h1>{i18n.gettext('Page not found')}</h1>
+          <p>
+            {i18n.gettext(oneLine`
+              Sorry, but we can't find anything at the address you entered.`)}
```

Comment:
I think 'URL' would be better than 'address'. It's more concrete and actionable.